src/ImageSharp/Formats/Tiff/PhotometricInterpretation/TiffColorDecoderFactory{TPixel}.cs

```diff
@@ -57,6 +57,16 @@ public static TiffBaseColorDecoder<TPixel> Create(TiffColorType colorType, ushor
                     DebugGuard.IsTrue(colorMap == null, "colorMap");
                     return new RgbTiffColor<TPixel>(bitsPerSample);
 
+                case TiffColorType.Rgb222:
+                    DebugGuard.IsTrue(
+                        bitsPerSample.Length == 3
+                        && bitsPerSample[0] == 2
```

Comment:
You can use the indexer check skipping trick discussed the other day here I reckon.